UI/js/common.js

```diff
@@ -74,15 +76,38 @@ const errors = Object.freeze({
     2007: 'Failed to update user, missing information',
     2008: 'invalid profile picture extension',
 
-    //3000 settings
+    //3000 settings errors
     3005: 'could not update the selected mode',
     3006: 'invalid mode',
     3007: 'website is not active',
 
-    //4000 custom file system
-    4010: 'permission denied'
+    //4000 custom file system errors
+    4010: 'permission denied',
+
+    defaultError: 'Something went wrong, please try again!',
+    passwordsDontMatch: 'Passwords do not match',
+    uploadOnlyPicture: 'You can only upload one picture!',
+    noResultsFoundBasedOnSearch: 'No results found based on your search',
+    mustBeCsv: 'You can only import one file!',
+    mustImportOneFile: 'File format must be csv!',
+    successfulFileUpload: 'File uploaded successfully',
+
```

Comment:
Should we leave this here, with translations and all planned?

Reply:
this is the UI translation object, so it will contain any message that will be shown to the user. as for the format, I talked to Ramy about this, and he will fix it in his next PR since it wont be an simple fix